src/assets/carousel.css

```diff
@@ -0,0 +1,93 @@
+/* GLOBAL STYLES
+-------------------------------------------------- */
+/* Padding below the footer and lighter body text */
+
+body {
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+  color: #5a5a5a;
+}
+
+
+/* CUSTOMIZE THE CAROUSEL
+-------------------------------------------------- */
+
+/* Carousel base class */
+.carousel {
+  margin-bottom: 4rem;
+}
+/* Since positioning the image, we need to help out the caption */
+.carousel-caption {
```

Comment:
Since this branch is related to the footer, I didn't make components that were related to the layout in general. How should I proceed to make the changes necessary for the layout with the already made footer?

Reply:
Just add them as scoped styles in the `Sample.vue` where I presume all the styling is needed (you should be able to cut and paste it without any alterations)